bin/make-ali-ami

```diff
@@ -5,10 +5,10 @@ VERSION=43
 region=eu-central-1
 aliyun ecs ImportImage \
  --Description="gardenlinux-dev-ali-$VERSION" \
- --Platform=Debian \
+ --Platform='Others Linux' \
```

Comment:
just from the issue discription I saw you quoting `OtherLinux` instead, does wording matter, or any other random value other than `Debian` would do that job?

Reply:
hmm infact i think we are a debian ... from the technical site ... so i do not know if the implications of "Others Linux" are bad for us

Reply:
> just from the issue discription I saw you quoting `OtherLinux` instead, does wording matter, or any other random value other than `Debian` would do that job?

It should be `Others Linux`. It is a default value. If it is set Debian, it will fail to bootstraped within desired time. Alicloud inject something for different OS types. e.g. I tried to set type `CoreOS`, it could be booted quickly but failed in ssh connection.

Reply:
> hmm infact i think we are a debian ... from the technical site ... so i do not know if the implications of "Others Linux" are bad for us

I could check with Alicloud

Reply:
hmm it is an official debian we have nothing from extern - all debian. BUT we change the default behavior e.g. debian has some stuff that is systemd and some is not -> networking is not systemd based it is the old dhclient stuff -> that we switched to pure systemd

Reply:
but all in all it is a debian

Reply:
I am sue it has nothing to do with that. Alicloud should explain what the mean by "Debian" and "other" and what precisely takes 20 minutes. Maybe they try some optimisations for the 4.x kernel that fails because we use a 5.4 kernel !?

Reply:
I have already created another ticked to Alicloud. Also put all these pieces of information to the ticket.

Reply:
Just got the reply:

> IF you use "Platform=“Others Linux”" on Debian image, it will cause some default setting being unset, such as software source, ntp, network, etc.. Thank you.

Reply:
I don't understand. What will be unset. NTP server is configured in the image, network is handled by dhcp, there is no software source ...

Reply:
I did a test for Image Platform=“Others Linux” and make it running in Alicloud Gardener cluster, it works.